server/index.js

```diff
@@ -2,17 +2,41 @@ const express = require('express');
 const db = require('../database/index');
 
 const app = express();
-const port = process.env.PORT;
-if (port === null || port === '') {
-  port = 8000;
-}
+var port = process.env.PORT;
 
-// app.get('/', (req, res) => res.send('Howdoo'));
+if (!port || port === '') {
+  port = 3000;
+}
 
+// test out sending all rows from trail table
 app.get('/', (req, res) => {
   db.getAllTrails( (rowsRes) => {
     res.json(rowsRes);
   });
 });
 
+app.get('/:trailId/trailInfo', (req, res) => {
+  var theId = req.params.trailId;
+  db.getAtrail(theId, (row) => {
+    // TODO
+    // trail object for response
+    var theTrail = row[0];
+    var resObj = {};
+    resObj.data = {};
+    resObj.data.attributes = {};
+    for (var prop in theTrail) {
+      if (prop === 'trail_id') {
+        resObj.data['trail_id'] = theTrail['trail_id'];
+      } else {
+        resObj.data.type = 'trail';
+        resObj.data.attributes[prop] = row[0][prop];
```

Comment:
Hey Jeff, just poking around. 

On line 32 there's row[0]. Any way this can not be hard coded? If not, (I'm thinking that it's not, just checking) it's the same as theTrail on line 32, right?